src/components/Search/SearchPageHeader/SearchFiltersBar.tsx

```diff
@@ -425,6 +435,7 @@ function SearchFiltersBar({queryJSON, headerButtonsOptions, isMobileSelectionMod
                           PopoverComponent: postedPickerComponent,
                           value: displayPosted,
                           filterKey: FILTER_KEYS.POSTED_ON,
+                          dateFilterKey: CONST.SEARCH.SYNTAX_FILTER_KEYS.POSTED,
```

Comment:
Instead of adding this key, can we just use the this array https://github.com/Expensify/App/blob/225e37c6bab29f24634b6ce1fbd118bdd8b1114b/src/types/form/SearchAdvancedFiltersForm.ts#L7-L15, so we don't have to maintain this in two separate places?